dev/PushNotifications/PushNotificationReceivedEventArgs.h

```diff
@@ -17,9 +19,17 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         bool Handled();
         void Handled(bool value);
 
+        byte* GetByteArrayFromBuffer(winrt::Windows::Storage::Streams::IBuffer buffer);
+
     private:
-        const winrt::Windows::Storage::Streams::IBuffer m_rawNotification{};
+
+        byte* m_rawNotification = nullptr;
+
         const winrt::Windows::ApplicationModel::Background::IBackgroundTaskInstance m_backgroundTaskInstance{};
         const winrt::Windows::Networking::PushNotifications::PushNotificationReceivedEventArgs m_args = nullptr;
+
+        ULONG m_length;
```

Comment:
This should be deleted. Store the bytes in a vector please and infer length from the vector instead.